clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-12-vexxhost-hybrid_clusterpool.yaml

```diff
@@ -37,17 +44,17 @@ spec:
   - name: openstack-k8s-operators-cdc-vexx-23
   labels:
     tp.openshift.io/owner: openstack-k8s-operators
-  maxConcurrent: 14
-  maxSize: 14
+  maxConcurrent: 21
+  maxSize: 21
```

Comment:
how are we deciding these values

Reply:
@marios 

maxSize: We determine size from our inventory size, we have created  floating ip manually/entry in AWS and created CDC files for 20 clusters: https://github.com/openshift/release/blob/a8ec2ee966b50045082d6888172483810da1798f/clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-cdc-vexx-01.yaml ) for 20 OCP clusters.

~~~
  - name: openstack-k8s-operators-cdc-vexx-01
  - name: openstack-k8s-operators-cdc-vexx-02
  - name: openstack-k8s-operators-cdc-vexx-03
  - name: openstack-k8s-operators-cdc-vexx-04
  - name: openstack-k8s-operators-cdc-vexx-05
  - name: openstack-k8s-operators-cdc-vexx-06
  - name: openstack-k8s-operators-cdc-vexx-07
  - name: openstack-k8s-operators-cdc-vexx-08
  - name: openstack-k8s-operators-cdc-vexx-09
  - name: openstack-k8s-operators-cdc-vexx-10
  - name: openstack-k8s-operators-cdc-vexx-11
  - name: openstack-k8s-operators-cdc-vexx-12
  - name: openstack-k8s-operators-cdc-vexx-13
  - name: openstack-k8s-operators-cdc-vexx-14
  - name: openstack-k8s-operators-cdc-vexx-15
  - name: openstack-k8s-operators-cdc-vexx-16
  - name: openstack-k8s-operators-cdc-vexx-17
  - name: openstack-k8s-operators-cdc-vexx-18
  - name: openstack-k8s-operators-cdc-vexx-19
  - name: openstack-k8s-operators-cdc-vexx-20
  ~~~
  
  runningCount is how many clusters we want in ready state as standby(ready to be claimed anytime)

Reply:
thank you for the pointers @Sandeepyadav93